base_report_to_printer/models/res_users.py

```diff
@@ -23,6 +23,9 @@ def _user_available_action_types(self):
     printing_printer_id = fields.Many2one(
         comodel_name="printing.printer", string="Default Printer"
     )
+    default_label_printer_id = fields.Many2one(
```

Comment:
That new field should also be added in `_register_hook` below.
We must allow the user to set a value in his preferences